### DISCUSSION TITLE
What's the intended way of protecting uploaded documents?

### DISCUSSION THREAD

**Discussion by ronny-rentner** [docusealco/docuseal#363] [Q&A] [answered]
I am using the Docker container and when I've uploaded a document there, it seems to be available to the world with no protection if you know the URL. Is there a setting somewhere that I haven't found?

**Comment by omohokcoj** [comment]
@ronny-rentner could you please share the URL you want to be disabled?

**Comment by ronny-rentner** [comment]
They're all in the form `https://localhost:3000/d/somehash123`

**Reply by omohokcoj** [reply, marked as answer]
@ronny-rentner currently that's not possible to disable the form links - but that's something we might add in the future.
Archiving the template should disable those links, also to view the documents via this link the template should contain only one signing party and the email should be provided so that you can review the submitted emails on the template page.
Alternatively, you can block /d/ path in your reverse proxy configurations.

**Reply by ronny-rentner** [reply]
Oh, ok, I wasn't aware that the behavior might change with multiple signing parties. I'll try that. Thanks for your quick response.
